tic_tac_toe/game.py

```diff
@@ -47,7 +55,7 @@ def _is_winning_combination(board, combination, player):
     Returns True of all three positions in the combination belongs to given
     player, False otherwise.
     """
-    pass
+    return not False in [board[i][j] == player for i,j in combination]
```

Comment:
It's a bit confusing but the `not` is negating the entire `False in [board[i][j] == player for i,j in combination]` statement. Basically we want to return True if every element in `[board[i][j] == player for i,j in combination]` is True. It would probably be more readable if we used all()
`return all([board[i][j] == player for i,j in combination])`